template.yaml

```diff
@@ -9,9 +9,6 @@ Description: >
 Globals:
   Function:
     Timeout: 5
-    Environment:
-      Variables:
-        firestoreCred:
```

Comment:
You can pass some dummy value here, instead of leaving it empty?

Reply:
sure
Done please re-review